src/main/java/com/denizenscript/ddiscordbot/properties/DiscordElementTagExtensions.java

```diff
@@ -0,0 +1,28 @@
+package com.denizenscript.ddiscordbot.properties;
+
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.objects.core.TimeTag;
+import net.dv8tion.jda.api.utils.TimeUtil;
+
+public class DiscordElementTagExtensions {
+
+    public static void register() {
+
+        // <--[tag]
+        // @attribute <ElementTag.discord_id_to_time>
+        // @returns TimeTag
+        // @group math

```

Comment:
This is missing `// @Plugin dDiscordBot`, and the `group` should be `extensions`